web_editor/main.css

```diff
@@ -33,6 +35,13 @@
   width: 50px;
 }
 
+.insert-lane {
+  position: absolute;
+  right: -15px;
```

Comment:
![Screenshot from 2022-11-07 11-50-38](https://user-images.githubusercontent.com/1664407/200313912-8be00957-06f9-412c-8b0d-f88ae10156fd.png)
Actually, I got the icons to be in a better place. Is this a reasonable way of achieving it?

Reply:
Perfect use of `position: absolute`! We want to anchor it to the bottom right of the card and that's idiomatic CSS to do so.